habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -279,7 +274,11 @@ def _update_markers(self) -> None:
             m.update()
 
     @add_perf_timing_func()
-    def reset(self):
+    def reset(self) -> None:
+        """
+        Reset the Simulator instance.
+        NOTE: this override does not return an observation.
```

Comment:
Note something that this PR introduces, but wondering if here we should return some empty observation or a format that is compatible with the rest of resets.

Reply:
Good question, by default python handles None, so I'm assuming the code is already robust against that given this code is fairly old. Honestly, why should reset return an observation anyway? Seems like something we should be asking for when we need it, not something provided without choice.
The top level reset or the Env reset can provide an observation if it is so important to have one for RL library compatibility.